File: MT.Model/Business/ProjectSurvey.cs
using System;
using System.ComponentModel.DataAnnotations;

namespace MT.Model
{
    #region ProjectSurvey:EntityBase
    /// <summary>
    /// This object represents the properties and methods of a ProjectSurvey:EntityBase.
    /// </summary>
    public class ProjectSurvey : EntityBase
    {
        [Key]
        public Guid ProjectSurveyID { get; set; }
        public Guid ProjectID { get; set; }
        public string FileName { get; set; }
        public string FileType { get; set; }
        public double FileSize { get; set; }
        public bool Inactive { get; set; }
        public int SortOrder { get; set; }

        public Guid FileResourceID { get; set; }
    }
    #endregion
}

File: BECore/app/model/ProjectSurvey.js
/// <summary>
/// Model EXTJS ProjectSurvey.
/// </summary>
Ext.define('QLDT.model.ProjectSurvey', {
    extend: 'Ext.data.Model',
    idProperty: 'ProjectSurveyID',
    idType: 'Guid',
    fields: [
        { name: "ProjectSurveyID", type: "string" },
		{ name: "ProjectID", type: "string" },
		{ name: "FileName", type: "string" },
		{ name: "FileType", type: "string" },
		{ name: "FileSize", type: "string" },
		{ name: "Inactive", type: "boolean" },
		{ name: "SortOrder", type: "int" },
		{ name: "CreatedDate", type: "date" },
		{ name: "ModifiedDate", type: "date" },
		{ name: "IPAddress", type: "string" },
		{ name: "ModifiedBy", type: "string" },
		{ name: "CreatedBy", type: "string" },
    ]
});

